LambdaEngine/Source/Game/ECS/Systems/Player/PlayerMovementSystem.cpp

```diff
@@ -39,19 +39,51 @@ namespace LambdaEngine
 
 	void PlayerMovementSystem::FixedTick(Timestamp deltaTime)
 	{
-		int8 deltaForward = int8(Input::IsKeyDown(EKey::KEY_T) - Input::IsKeyDown(EKey::KEY_G));
-		int8 deltaLeft = int8(Input::IsKeyDown(EKey::KEY_F) - Input::IsKeyDown(EKey::KEY_H));
+		int8 deltaForward	= int8(Input::IsKeyDown(EKey::KEY_T) - Input::IsKeyDown(EKey::KEY_G));
+		int8 deltaLeft		= int8(Input::IsKeyDown(EKey::KEY_F) - Input::IsKeyDown(EKey::KEY_H));
 
 		for (Entity entity : m_ControllableEntities)
 		{
-			Move(entity, deltaTime, deltaForward, deltaLeft);
+			ECSCore* pECS = ECSCore::GetInstance();
+			auto* pInterpolationComponents	= pECS->GetComponentArray<ControllableComponent>();
```

Comment:
pControllableComponents
```suggestion
			auto* pControllableComponents = pECS->GetComponentArray<ControllableComponent>();
```